Dev netpyne updates

filippomc opened the pull request.

pgleeson committed:
Test build OSB forrk/branch

pgleeson committed:
Update requirements for latest netpyne and neuroml libraries

pgleeson committed:
Quicker download of head model

pgleeson committed:
Merge pull request #992 from OpenSourceBrain/develop

Develop to netpyne branch

filippomc committed:
Revert change netpyne branch

filippomc committed:
Merge branch 'develop' into dev_netpyne_updates

filippomc commented on applications/netpyne/Dockerfile:
```diff
@@ -16,7 +16,9 @@ RUN yarn build-dev
 USER root
-RUN wget --no-check-certificate -O /nyhead.mat https://www.parralab.org/nyhead/sa_nyhead.mat
+RUN apt-get update -qq && apt-get install aria2 -y
+RUN aria2c -j10 -x 10 https://www.parralab.org/nyhead/sa_nyhead.mat
```
download and move are better done in one single command, as otherwise the file is present in 2 separate layers, increasing the image size

pgleeson replied:
Ok, have made that change. Use of aria2c seems to increase the speed of downloading, sometimes wget would be stuck for ages with the download.

pgleeson committed:
Consolidated download of nyhead.mat into one line

pgleeson committed:
Allow a local directory, shared, to be accessed inside the docker container for netpyneui

filippomc committed:
Remove netpyne gatekeeper

filippomc committed:
Merge branch 'dev_netpyne_updates' of github.com:OpenSourceBrain/OSBv2 into dev_netpyne_updates

pgleeson committed:
Merge branch 'dev_netpyne_updates' of github.com:OpenSourceBrain/OSBv2 into dev_netpyne_updates

pgleeson committed:
Minor changes in Dockerfile to aid admin bash scripting

pgleeson committed:
Update About box: Add Gleeson lab, full url to logos

Url update for https://github.com/OpenSourceBrain/OSBv2/issues/990. This
may not be an issue on live, so could be reverted...

filippomc committed:
Patch keycloak bootstrap issue

sanjayankur31 committed:
feat(local-script): do not make a skaffold failure stop minikube

pgleeson committed:
Updates to Jupyterlab: uses osbv2 netpyne branch & add scripts

filippomc committed:
dev pipeline update

filippomc committed:
Merge branch 'dev_netpyne_updates' of github.com:OpenSourceBrain/OSBv2 into dev_netpyne_updates

pgleeson committed:
Use  --no-check-certificate for head model

pgleeson committed:
Merge branch 'develop' into dev_test_pg

pgleeson committed:
Merge branch 'dev_netpyne_updates' into dev_test_pg

pgleeson committed:
Merge pull request #1002 from OpenSourceBrain/dev_test_pg

Latest dev to netpyne test branch
